tc/core/polyhedral/cuda/mapped_scop.cc

```diff
@@ -649,6 +652,9 @@ std::unique_ptr<MappedScop> MappedScop::makeWithOuterBlockInnerThreadStrategy(
     scop->specializeToContext();
   }
 
+  // 9. Insert mapping context
```

Comment:
This change is independent of dependence computation and can be moved to a separate commit.  That would provide space to motivate the change.

Also, let's please change the numbering in the comment and use 5 here and 6+ below.